### PR TITLE
Update documentation to work with rtd

### PR DESCRIPTION
**Pull request by jwbishop**
Documentation files updated for read the docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,12 +46,19 @@ See the included `Example_Processing.py`.
 ## References and Credits ##
 If you use this code for array processing, we ask that you cite the following papers:
 
-1. Bishop, J.W., Fee, D., & Szuberla, C. A. L., (in review). Improved infrasound array processing with robust estimators, Geophysical Journal International
+1. Bishop, J.W., Fee, D., & Szuberla, C. A. L., (2020). Improved infrasound array processing with robust estimators, Geophys. J. Int., Vol(num) p. ###-### doi: [https://doi.org/10.1093/gji/ggaa110](https://doi.org/10.1093/gji/ggaa110)
 
 2. Rousseeuw, P. J. & Van Driessen, K., 2006. Computing LTS regression for large data sets, Data Mining and Knowledge Discovery, 12(1), 29-45
+doi: [10.1007/s10618-005-0024-4](10.1007/s10618-005-0024-4)
```
Comment (liamtoney) [inline README.md]: This link doesn't work.

**Review by liamtoney — COMMENTED** [review]
I'd recommend including the pre-existing README on your RTD site instead of creating a new version. Check out e.g. _array_processing_ to see how this is done

—

EDIT: Oh, I see. You have two versions, I guess you could just remove README.md in favor of README.rst?

File: docs/index.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Welcome to lts_array's documentation!
+=====================================
+
+.. toctree::
+   :caption: README:
+
+   README.rst
```
Comment (liamtoney) [inline docs/index.rst:7]: I think — if you were to go with the strategy of having the README.rst in the top level of the repo — that you could use this and remove `docs/README.rst`. You'd have to test though, I'm not positive
```suggestion
   ../README.rst
```

**Review by liamtoney — CHANGES_REQUESTED** [review]
Unless you are gonna have folks on Windows **building** the docs themselves (I think unlikely) you can remove `doc/make.bat` since that's the Windows version of a `Makefile`

File: README.md (hunk 1 above)
Comment (jwbishop) [reply README.md]: Fixed! Good catch.

File: docs/index.rst (hunk 2 above)
Comment (jwbishop) [reply docs/index.rst:7]: This doesn't work. The in-docs README is a command to `.. include::` the actual README on the level up. Either adding the README on the level up or adding the include statement to the index file results in Sphinx being unable to find the file.

**Comment by jwbishop** [comment]
> I'd recommend including the pre-existing README on your RTD site instead of creating a new version. Check out e.g. _array_processing_ to see how this is done
> 
> 
> EDIT: Oh, I see. You have two versions, I guess you could just remove README.md in favor of README.rst?

I could, but I think I'm going to leave both for now, and potentially remove the .md file at a later release. The Pandoc conversion of rst to md is imperfect, and this would (at least temporarily) avoid making edits to the rst file itself. I also want to see if users end up looking at the included README.

**Comment by jwbishop** [comment]
> Unless you are gonna have folks on Windows **building** the docs themselves (I think unlikely) you can remove `doc/make.bat` since that's the Windows version of a `Makefile`

Good catch. Removed.

**Comment by liamtoney** [comment]
> The Pandoc conversion of rst to md is imperfect, and this would (at least temporarily) avoid making edits to the rst file itself.

Hmm. What command did you use? I had good success with
```
pandoc -f markdown -t rst README.md -o README.rst
```
Anyways, not a big deal

File: docs/index.rst (hunk 2 above)
Comment (liamtoney) [reply docs/index.rst:7]: Ahh, yes. I forgot that you'd need to use this trick: https://stackoverflow.com/a/17217041

Comment (liamtoney) [reply docs/index.rst:7]: Whoops, I'm sorry. Looks like you've already done this

**Review by liamtoney — COMMENTED** [review]
This looks good to me — except that I'm still confused about the need to retain README.md. The formatted output looks identical to me.

**Comment by jwbishop** [comment]
> This looks good to me — except that I'm still confused about the need to retain README.md. The formatted output looks identical to me.

After going to/from .rst with Pandoc, the conversion looks better now. The tool I had been using was ok, but Pandoc is better. I now feel comfortable removing the .md file.

**Review by liamtoney — APPROVED** [review]
Great! I think this is good to go!

**Review by davidfee5 — APPROVED** [review]
Looks good to me. Nicely done!

**Pull request merged**